tools/replace_code_block.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+from traceback import print_tb
+from typing import AnyStr
+
+status = 0
+
+
+def check_for_code(file: AnyStr, check: bool):
```

Comment:
Since we're passing in a string, could we name this `filename`, or `filepath`?

Reply:
I guess filename makes more sense